v2/parser/parse.go

```diff
@@ -510,11 +516,13 @@ func (p *Parser) addPkgToUniverse(pkg *packages.Package, u *types.Universe) erro
 
 	// Add all of this package's imports.
 	importedPkgs := []string{}
-	for _, imp := range pkg.Imports {
-		if err := p.addPkgToUniverse(imp, u); err != nil {
+	// Iterate imports in a predictable order
+	for _, imp := range slices.Sorted(maps.Keys(pkg.Imports)) {
+		pkg := pkg.Imports[imp]
```

Comment:
avoid shadowing pkg, keep `imp` referring to the same thing?

```
// Iterate imports in a predictable order
for _, key := range slices.Sorted(maps.Keys(pkg.Imports)) {
  img := pkg.Imports[key]
```

Reply:
Done